packages/schematics/angular/utility/find-module.ts

```diff
@@ -42,8 +43,9 @@ export function findModuleFromOptions(host: Tree, options: ModuleOptions): Path
   const routingModuleExt = options.routingModuleExt || ROUTING_MODULE_EXT;
 
   if (!options.module) {
+    options.normalizeName = options.normalizeName || strings.dasherize;
```

Comment:
Instead of allowing for overwriting the name, why don't you let the parent pass in the formatting function (with dasherize being the default)? We have [`strings.underscore()`](https://github.com/angular/angular-cli/blob/master/packages/angular_devkit/core/src/utils/strings.ts#L106) which would allow you to format the name properly in google, whichever the input. What do you think?

Reply:
Ugh Sorry. I think the variable is badly named. This is exactly what I'm trying to do.

  normalizeName?: (str: string) => string;

Renamed it to nameFormatter.